### PR TITLE
Check for valid current reference to add or remove event listeners

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:** 
#476

**Description of changes:**
- Checks for validity of elements current ref in `useFocusIn` and `useMouseMove` while adding / removing event listeners.
- Removes snapshot testing workflow.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes.

2. How did you test these changes?
- Run the `demo/meeting` app.
- Join a meeting.
- Go away from the screen until the meeting control (options of mic, speaker etc) disappear.
- Come back in, they should re-appear now.
- Leave the meeting and checked and found no browser console errors related to `useFocusIn`, `useFocusOut` and `useMouseMove`.

3. If you made changes to the component library, have you provided corresponding documentation changes? NA.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/hooks/useFocusIn/index.tsx [hunk 1]
```diff
@@ -24,15 +24,15 @@ export function useFocusIn(el: RefObject<any>, delay = 3000) {
       }, delay);
     };
 
-    el.current.addEventListener('focusin', onFocusIn);
-    el.current.addEventListener('focusout', onFocusOut);
+    el.current?.addEventListener('focusin', onFocusIn);
```
Comment (ltrung) [inline src/hooks/useFocusIn/index.tsx:27]: Is this needed? Given that we already return in line 11. I think this is just needed for the removeEventListener

**Review by ltrung — COMMENTED** [review]
Can you double check the changes in package-lock.json? Seem like a lot of changes so make sure it is not reverting back to v1.

**Review by ltrung — COMMENTED** [review]
Can you add the steps on how you tested the apps and verify there is no console error log?

File: src/hooks/useFocusIn/index.tsx (hunk 1 above)
Comment (devalevenkatesh) [reply src/hooks/useFocusIn/index.tsx:27]: Yeah right not needed thanks for the catch!

**Comment by devalevenkatesh** [comment]
> Can you double check the changes in package-lock.json? Seem like a lot of changes so make sure it is not reverting back to v1.

Yup, have verified they are `v2` changes only.

**Comment by devalevenkatesh** [comment]
> Can you add the steps on how you tested the apps and verify there is no console error log?

Yup done.

**Comment by ltrung** [comment]
> > Can you double check the changes in package-lock.json? Seem like a lot of changes so make sure it is not reverting back to v1.
> 
> Yup, have verified they are `v2` changes only.

Just curious what are those changes? Seem like a lot of changes for a couple line changes.

**Comment by devalevenkatesh** [comment]
> > > Can you double check the changes in package-lock.json? Seem like a lot of changes so make sure it is not reverting back to v1.
> > 
> > 
> > Yup, have verified they are `v2` changes only.
> 
> Just curious what are those changes? Seem like a lot of changes for a couple line changes.

So, the lockfile version is still v2, but I had deleted node_modules and package-lock.json and regenerated them while testing and running the demo meeting app. Hence, I think a lot of dependency updates in the package-lock.json. I have anyways removed that file from the commit.

**Review by ltrung — APPROVED** [review]

**Pull request merged**